components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -300,17 +305,97 @@ class Queue(BaseModel):
     host: str = "localhost"
     port: int = 5672
 
-    username: typing.Optional[str]
-    password: typing.Optional[str]
+    username: Optional[str]
+    password: Optional[str]
 
 
-class ArchiveOutput(BaseModel):
+class S3Config(BaseModel):
+    region_name: str
```

Comment:
I am using region_name since it follows the boto3 parameter name, and in boto3, region_name does refere to names like "us-west-1".

But sure, we can change here to region_code to be consistent with aws.